pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -706,7 +706,7 @@ def test_add_image_features():
             "Metadata_Well": ["A01"],
             "Metadata_Site_Count": [2],
             "Metadata_Object_Count": [100],
-            "Image_Count_Cells": [100],
+            "Metadata_Count_Cells": [100],
```

Comment:
This might be more of a philosophical question that I'm sure JUMP has talked about already. Those discussions probably already happened, which warranted this PR! (my guess at least)

Is "count" metadata or part of the profile?

More generally, how do you know if something is metadata or not? 🤔 

I think it is totally fine as is, just an interesting thought

Reply:
Great question. I thought about it for sometime before creating this PR but eventually made the decision that it is a metadata column so that I can finish creating the final pipeline for JUMP :) The timeline of JUMP often influences that decisions I make. But this is definitely a question that needs to be thought about when there is more time to ponder.